src/ducks/modules/stage.js

```diff
@@ -1,13 +1,18 @@
+import { actionTypes as protocolActionTypes } from '../../ducks/modules/protocol';
+
 const NEXT_PROMPT = 'NEXT_PROMPT';
 const PREVIOUS_PROMPT = 'PREVIOUS_PROMPT';
+const NEXT_STAGE = 'NEXT_STAGE';
+const PREVIOUS_STAGE = 'PREVIOUS_STAGE';
 
 const initialState = {
   promptIndex: 0,
```

Comment:
Yeah you're probably right. It's here because it's reset by a couple of actions, but then it only needs to be reset because it's here... Could just store it using setState on the component. That said, it doesn't hurt it being stored in the redux store, and means that the rehydrate step remembers which prompt you are on.

I'm not totally happy with the name of this module either, any thoughts? `particpantProgress`?